src/ui/common/src/components/layouts/menuSidebar.tsx

```diff
@@ -123,22 +123,19 @@ const MenuSidebar: React.FC<{
   const [versionNumber, setVersionNumber] = useState('');
   const location = useLocation();
 
+  const { data } = useEnvironmentGetQuery({ apiKey: user.apiKey } as any, {
```

Comment:
no need to cast to `any` and you probably don't need the `skip` here, as the user is passed by caller